api/axis.py

```diff
@@ -320,7 +320,7 @@ def _checkout(self, title_id, patron_id, internal_format):
         response = self.request(url, data=args, method="POST")
         return response
 
-    def fulfill(self, patron, pin, licensepool, format_type, **kwargs):
+    def fulfill(self, patron, pin, licensepool, **kwargs):
```

Comment:
Axis 360 can completely ignore this argument because the format type is locked in on checkout.

Reply:
What do you mean that it's `locked in`? Is the type (like ebook) set before it's fulfilled?

Reply:
Yeah, for Axis 360 you choose the content type (EPUB or PDF) as part of taking out the loan. With Overdrive or Bibliotheca you choose the content type the first time you ask for an ACSM file.

In either case, once you make your choice you're locked in for the duration of the loan.